install_config/persistent_storage/persistent_storage_ceph_rbd.adoc

```diff
@@ -115,13 +166,92 @@ ceph-pv                  <none>    2147483648   RWO           Available
 ----
 ====
 
-Users can then link:../../dev_guide/persistent_volumes.html[request storage
-using persistent volume claims], which can now utilize your new persistent
-volume.
+Developers request Ceph-RBD storage by referencing, in the `*volumes*` section
+of their pod spec, either a
+link:../../dev_guide/persistent_volumes.html[`*PersistentVolumeClaim*`] by name,
+or the gluster volume plug-in directly. A PVC exists only in the user's namespace
+and can only be referenced only by pods within that same namespace. Any attempt to
+access a PV across a namespace causes the pod to fail.
 
-[IMPORTANT]
+The next step can be to create a persistent volume claim (PVC) which will bind
+to the new PV:
+
+.PVC Object Definition
 ====
-Persistent volume claims only exist in the user's namespace and can only be
-referenced by a pod within that same namespace. Any attempt to access a
-persistent volume from a different namespace causes the pod to fail.
+[source,yaml]
+----
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: ceph-claim
+spec:
+  accessModes: <1>
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 2Gi <2>
+
+----
+<1> As mentioned above for PVs, the `*accessModes*` do not enforce access right,
+but rather act as labels to match a PV to a PVC.
+<2> This claim will look for PVs offering *2Gi* or greater capacity.
+====
+
+Save the definition to a file, for example *ceph-claim.yaml_*, and create the
+PVC:
+
+====
+----
+# oc create -f ceph-claim.yaml
+----
+====
+
+[[ceph-volume-security]]
+=== Volume Security
+
+This section covers Ceph-RBD block volume security.
+
+[NOTE]
+====
+See
+link:../../install_config/persistent_storage/pod_security_context#fsgroup.html[Volume
+Security] before implementing ceph volumes.
+====
+
+A significant difference between _shared_ volumes (such as NFS or GlusterFS) and
+_block_ volumes (such as Ceph-RBD, iSCSI, most cloud storage) is that the UID
+and/or group IDs defined in the pod definition (or in the docker image) are applied
+to the *_target_* (physical) storage. This is referred to as *_taking over_* the
```

Comment:
Please say 'managing ownership of the block device' here and in all other similar docs.